CMSgov/design-system
# [NO-TICKET] Remove console.log, create eslint check for it

## Summary

While looking around the source, I found a stray `console.log` that is most likely leftover from recent debugging. This PR removes it and also adds an ESLint rule to fail use of any console method except console.warn and console.error in the production source code. It still allows logging in stories. 

Although this counts console.log as an error that will fail ESLint CI, it doesn't prevent compilation of log messages during local debugging. The goal is to prevent shipping code with debugging messages.

If this seems too complex, feel free to just remove the console.log and close this PR. 😸 

## How to test

Run the unit tests. You should no longer see the console message appear in the output.

## Checklist




### If this is a change to design:


### If this is a change to code:


### If this is a change to documentation:
